testdata/v2_atlantis_depends_on/fogg.yml

```diff
@@ -1,7 +1,8 @@
 defaults:
   backend:
     bucket: buck
-    profile: profile
+    role: role
+    account_id: "12345"
```

Comment:
this makes sure we hit the case where a role is used (which also requires `account_id` to be set)